examples/torque_driven_ocp/double_pendulum_with_translations.bioMod

```diff
@@ -48,57 +24,33 @@ endsegment
     // Marker 2
     marker marker_2
         parent Seg1
-        position 0.1 0.1 -1
+        position -0.0005 0.0688 -0.9542
     endmarker
 
-//Seg2
-    segment Seg2
+segment Seg2
     parent Seg1
-    rotations x
+    rotations	x
     rtinmatrix 0
-    rt 0 0 0 xyz 0.1 0.1 -1
-    ranges -pi/2 pi/2
+    rt 0 0 0 xyz -0.0005 0.0688 -0.9542
```

Comment:
This is not what I calculated by hand... 
I calculated `0 0 0 xyz 0 0 -1.1` If I recall well.

Reply:
what did you expected from the location in parent in this double pendulum ? I hanged the second segment at the com of the first segment like the old one.

Reply:
The CoM is inside of the Pendulum, it does not make sense to attach it at that point. I was expecting it to be aligned with the previous stick but right under the hammer

Reply:
ok then !